VirtoCommerce.Platform.Tests/UnitTests/ScriptCollectorTest.cs

```diff
@@ -0,0 +1,225 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Reflection;
+using Moq;
+using VirtoCommerce.Platform.Core.Modularity;
+using VirtoCommerce.Platform.Core.ModuleFileCollector;
+using VirtoCommerce.Platform.Core.VersionProvider;
+using Xunit;
+
+namespace VirtoCommerce.Platform.Tests.UnitTests
+{
+    public class ScriptCollectorTest
+    {
+        private string _destinationPath { get; set; }
+        private string _temporaryFolderName { get; set; }
+        private string _fullPhysicalPathToDistFolder { get; set; }
+
+        [Fact]
+        public void TestCollectorWithOneModuleWithAppAndVendorNoVersionAppend()
+        {
+            CreateTemporaryFiles(new[] { "app.js", "vendor.js" });
+
+            var fileVersionProvider = CreateMockedFileVersionProvider();
+            var localModuleCatalog = CreateMockedLocalModuleCatalog();
```

Comment:
It would be more convenient to create these mocks in the test class constructor.